src/frontend/komponenter/Behandling/Brev/Htmlfelter.tsx

```diff
@@ -0,0 +1,42 @@
+import { AndelTilkjentYtelse, TilkjentYtelse } from './BrevTyper';
+import { formaterNullableIsoDato, formaterTallMedTusenSkille } from '../../../utils/formatter';
+
+export const delmalTilHtml = (
+    delmalNavn: string,
+    tilkjentYtelse?: TilkjentYtelse
+): { [htmlFeltNavn: string]: string } => {
+    switch (delmalNavn) {
+        case 'varierendeInntekt':
+            return { inntektsperioderHtml: lagInntektsperioder(tilkjentYtelse) };
+        case 'testHtml':
```

Comment:
Denne skal slettes her og i sanity så snart vi får tabell inn i varierende inntekt igjen.

Reply:
Hva betyr det i praksis for oss som er grønne?

Reply:
Jeg la til en delmal (alavarierende inntekt) i brevgeneratoren med navn "testHtml". Dette for å kunne teste tabell uten å gå i bena på Hanna som jobber videre med innhold på varierende inntekt. Denne kan slettes når testing er ferdig. Vi kan også slette kode: case 'testHtml': (...)